includes/discovery/sensors/frequency/junos.inc.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/*
+ * LibreNMS
+ *
+ * Copyright (c) 2016 Neil Lathwood <neil@lathwood.co.uk>
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ *tis*/
+
+echo 'JunOS ';
+
+$multiplier = 1000000;
```

Comment:
Instead of multiplying by 1000000 here, you should probably just store the value in MHz.
This also avoids having to change the sensor limits.

Reply:
Ignore this. :)  The value should be stored in Hz.  (Wireless sensors are stored in MHz)